packages/md-enhance/src/client/styles/playground.scss

```diff
@@ -0,0 +1,198 @@
+.playground-container {
+  overflow: hidden;
+
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid;
+  border-color: var(--code-border-color);
+  border-radius: 6px;
+
+  // stylelint-disable max-nesting-depth
+  @media (max-width: hope-config.$tablet) {
+    &.internal {
+      .op-btns {
+        .op-btn.code {
+          display: none;
+        }
+      }
+    }
+  }
+
+  .op-btns {
+    display: flex;
+    align-items: center;
+
+    // stylelint-disable selector-max-compound-selectors
+    .op-btn {
+      display: inline-flex;
+      margin-left: 10px;
+
+      .icon {
+        width: 1.5rem;
+        height: 1.5rem;
+        fill: #777;
+      }
+
+      // stylelint-disable max-nesting-depth
+      &:hover {
+        .icon {
+          transition: transform 0.15s cubic-bezier(0.2, 0, 0.13, 2);
+          transform: scale(1.2);
+        }
+      }
+      // stylelint-enable max-nesting-depth
+    }
+    // stylelint-enable selector-max-compound-selectors
+  }
+  // stylelint-enable max-nesting-depth
+
+  .title-container {
+    display: flex;
+    flex-flow: row wrap;
+
+    padding: 8px 10px;
+    border-bottom: 1px solid;
+    border-bottom-color: var(--code-border-color);
+
+    background-color: var(--code-demo-header-bg-color, #eee);
```

Comment:
As people may only enable playground, but not code demo, you may declare a new css variable here